tests/containers/bci_prepare.pm

```diff
@@ -103,6 +132,10 @@ sub run {
     }
     my $bci_virtualenv = get_var('BCI_VIRTUALENV', 0);
 
+    # Provide SCC credentials for non_SLE host running specific BCI tests
+    scc_credentials_host(($engines =~ /podman/) ? 'podman' : 'docker', get_required_var('SCC_REGCODE'))
+      if get_var('CONTAINER_IMAGE_TO_TEST') =~ /fips:15.[34]/i && $host_distri !~ /sle|micro|opensuse/i;
```

Comment:
This doesn't affect only FIPS images but also all LTSS images.

I would even go so far to say that this needs to be done on all non-SLES hosts, if `SCC_REGCODE` is provided.

Reply:
This option comes from evidence that the same openqa tests on non-sle host running `NON-fips` containers `pass` as it is now on `master`. Also I don't see errors on ubuntu running containers 15sp5-fips, that is `>15sp4`.

But if we want to be sure and extend the correction, I remove all conditions on CONTAINER_IMAGE_TO_TEST.